python/dask_cudf/dask_cudf/tests/test_sort.py

```diff
@@ -47,6 +47,17 @@ def test_sort_values(nelem, nparts, by, ascending):
     dd.assert_eq(got, expect, check_index=False)
 
 
+@pytest.mark.parametrize("by", ["b", ["a", "b"]])
+def test_sort_values_categorical_raises(by):
```

Comment:
Locally, this fails for me - since we're only checking if the first column is categorical, should this instead be:

```suggestion
@pytest.mark.parametrize("by", ["b", ["b", "a"]])
def test_sort_values_categorical_raises(by):
```

Reply:
Weird - I made that change (tests were passing locally for me), and not sure how I lost it. Thanks!